README.md

```diff
@@ -4,6 +4,10 @@ Models, types and enumerations used by Hypersyncs.
 
 ## Release Notes
 
+### 5.1.0
+
+- Upgrade TypeScript to version 5.5.4
```

Comment:
This seems incomplete.  There are more changes here than just the TypeScript change.